conf/app.routes

```diff
@@ -1,3 +1,7 @@
 # microservice specific routes
 
-GET        /hello-world             uk.gov.hmrc.essttpstubs.controllers.MicroserviceHelloWorldController.hello
+#POST       /eligibility/:regime/:id/error  uk.gov.hmrc.essttpstubs.controllers.EligibilityController.error(regime: TaxRegime, id)
+
+POST       /eligibility/:regime/:id/financials  uk.gov.hmrc.essttpstubs.controllers.EligibilityController.financials(regime: TaxRegime, id)
```

Comment:
That API doesn't correspond to what is described in spec?
What is the purpose of that? 
Have a look on that:
https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=DTDT&title=Eligibility+API